Spring-Boot/Spring-Boot-Student/src/test/java/tw/waterball/judgegirl/springboot/student/controllers/StudentControllerTest.java

```diff
@@ -290,6 +292,21 @@ void Given10StudentsAnd4Admins0_1_2_3_WhenGetAdminsWithSkip2Size2_ShouldRespondA
         assertEquals("admin3", students.get(1).name);
     }
 
+    @Test
+    void GivenStudentSignedUp_WhenDeleteTheStudent_ShouldSucceed() throws Exception {
+        StudentView student = signUpAndGetResponseBody(this.student);
```

Comment:
rewrite to `var student = signUpAndGetStudent(this.student);`

Do me a favor, help me rename the method `signUpAndGetResponseBody` to `signUpAndGetStudent`

Reply:
Could you tell me why use var keyword in there?

Reply:
Because it doesn't compromise the readability too much and its type does not matter here. Moreover, type-inference is helpful when we are going to change the type In the future, especially in the case of a DTO type.